Add highlight for links to guest tutorials and Teachable Machine videos (#1906)
CodingTrain/website: _Tutorials/index.md

Here you can find all tutorials made by Daniel Shiffman on TheCodingTrain.

> If you are searching for tutorials made by <highlight>guests</highlight>, you can check them out [here]({{ site.baseurl }}{% link _GuestTutorials/index.md %}).

Reviewer: For accessibility, we should not use generic terms like "here". People who browse the internet with a screen reader read lists of the links and so meaningful text should always be included. 

For more see: https://www.washington.edu/accessibility/links/ and https://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs

Author: The actual links are still called "here", it's only the texts "guests" and "teachable machine" which are highlighted

Reviewer: Apologies for confusion it's the use of "here" that is a problem. The accessible way to create a link is as follows:

```
If you are searching for tutorials made by <highlight>guests</highlight>, you can check them
out on [the guest tutorial page]({{ site.baseurl }}{% link _GuestTutorials/index.md %})
```

There's probably a shorter way to phrase the above to reduce redundancy too!

Author: How about:
```suggestion
For even more tutorials, you can visit the [guest tutorials page]({{ site.baseurl }}{% link _GuestTutorials/index.md %}).
```
And
```suggestion
If you are interested in creating your own machine learning models, have a look at the [Teachable Machine page]({{ site.baseurl }}{% link _TeachableMachine/index.md %}).
```
English is not my native language so I'm open to any critic.

Reviewer: @violetcraze would you mind taking a look at the above discussion and adjusting the language on the site? Thank you! ❤️

Reviewer 2: Looks great to me!
